ruby/util.py

```diff
@@ -0,0 +1,56 @@
+import tokenize
+import ast
+
+from io import BytesIO
+from typing import List, Optional, Union, Any
+
+
+def split_into_tokens(code: str) -> List[str]:
+    try:
+        tokens = list(tokenize.tokenize(BytesIO(code.encode('utf-8')).readline))[1:]
```

Comment:
The last two tokens also seem to be technical (NEWLINE and ENDMARKER), maybe we want to omit them too?

Reply:
NEWLINE seems to be important for the multiline snippets (as well as INDENT / DEDENT).
ENDMARKER can be omitted for sure, thanks.